specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -3520,6 +4340,14 @@
         "publicNetworkAccess": {
           "description": "State indicating whether public traffic are allowed or not for a static web app. Allowed Values: 'Enabled', 'Disabled' or an empty string.",
           "type": "string"
+        },
+        "databaseConnections": {
+          "description": "Database connections for the static site",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StaticSiteDatabaseConnectionOverviewARMResource"
```

Comment:
>StaticSiteDatabaseConnectionOverviewARMResource [](http://example.com/codeflow?start=35&length=47)

What's the difference between a 'StaticSiteDatabaseConnectionOverview' and a 'StaticSiteDatabaseConnectionOverviewARMResource'? Aren't they conceptually the same thing?

Reply:
Can you clarify this comment? In the most recent iteration as of your review, I have removed references to StaticSiteDatabaseConnectionOverview in favor of StaticSiteDatabaseConnectionOverviewARMResource.